_includes/metrics_widgets/pie-chart.html

```diff
@@ -1,4 +1,4 @@
-{% if link %}<a href="{{ '/' | url }}{{link}}">{% endif %}
+{% if link %}<a href="{% link link %}">{% endif %}
```

Comment:
That's confusing and I'm surprised there's no naming overwrite in this case...

Reply:
Same!